sale_order_volume/models/sale_order.py

```diff
@@ -93,5 +93,9 @@ def compute_product_category_volumes(self):
 
     def write(self, values):
         ret = super(SaleOrder, self).write(values)
+        # fixme : don't put this in write, because it does not update
+        # the volume by category directly when changing the SO lines.
+        # with Odoo 16 and the "invisible save" flow, it's disturbing.
+        # use a computed field instead.
         self.compute_product_category_volumes()
         return ret
```

Comment:
Does this suffice for merging?

Reply:
I removed all the "volume by category" feature, because it's not used by the client. The problem I mentionned is not present anymore in the stripped down version of the module.

Reply:
ah, the downside of reviewing chronologically per commit :sweat_smile: